src/Microsoft.AspNetCore.Mvc.Abstractions/Formatters/IFormatterExceptionPolicy.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Formatters
+{
+    /// <summary>
+    /// A policy which <see cref="IInputFormatter"/>s can implement to indicate if they want the body model binder
+    /// to handle all exceptions. By default, all default <see cref="IInputFormatter"/>s implement this interface and
+    /// have a default value of <see langword="true"/>.
+    /// </summary>
+    public interface IFormatterExceptionPolicy
```

Comment:
Discussed with @kichalla and also overheard by @rynowak that we should change the APIs to:

```c#
public interface IInputFormatterExceptionPolicy
{
	InputFormatterExceptionModelStatePolicy ExceptionPolicy { get; }
}

public enum InputFormatterExceptionModelStatePolicy
{
	AllExceptions = 0, // default
	MalformedInputExceptions = 1,
}


public class MvcOptions
{
	...
	public InputFormatterExceptionModelStatePolicy InputFormatterExceptionModelStatePolicy { get; set; }
}
```